projects/extension/Dockerfile

```diff
@@ -65,10 +66,9 @@ USER root
 
 RUN pip install --break-system-packages uv==0.5.20
 
-WORKDIR /pgai
+## install only our python dependencies
+COPY pyproject.toml uv.lock /py/
+RUN uv sync --directory /py --no-install-project --frozen
+ENV PATH="/py/.venv/bin:$PATH"
 
-## install our test python dependencies
-COPY pyproject.toml uv.lock requirements-lock.txt build.py /pgai
-COPY ai /pgai/ai
```

Comment:
Don't we still want to copy in all of the other files of `ai`?

Reply:
nah. The code in the ai module gets installed in `/usr/local/lib/pgai/<version>/ai`

Actually, I *think* all we need here are the dev/test dependencies.

Reply:
yeah. all we need are the dependencies from the dev group